addons/pkg-npm/jaxrs/src/main/java/org/commonjava/indy/pkg/npm/jaxrs/NPMContentAccessResource.java

```diff
@@ -145,13 +159,27 @@ public Response doHead(
     public Response doHead(
                     final @ApiParam( allowableValues = "hosted,group,remote", required = true ) @PathParam( "type" ) String type,
                     final @ApiParam( required = true ) @PathParam( "name" ) String name,
-                    final @PathParam( "packageName" ) String packageName, final @PathParam( "version" ) String version,
+                    final @PathParam( "packageName" ) String packageName,
+                    final @ApiParam( allowableValues = CROSS ) @PathParam( "version" ) String version,
```

Comment:
One comment here: I *think* this will make the Swagger UI present the user with a drop-down containing only `-`. We should check that, though. It's not critical, but it's something we should check and adjust when time allows.